PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.java

```diff
@@ -52,30 +54,30 @@
     private JPanel clusterPanel;
     private JPanel authenticationPanel;
     private JPanel comboBoxPanel;
-    private JComboBox clusterComboBox;
-    private JPanel clusterCardsPanel;
+    protected JComboBox clusterComboBox;
```

Comment:
Are these `protected` changes necessary?

Reply:
Refer to #2410

---
In reply to: [237773385](https://github.com/Microsoft/azure-tools-for-java/pull/2235#discussion_r237773385) [](ancestors = 237773385)